tests/test_choice.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from gradebook import Gradebook
+class FakeInstructor:
+    def has_access(self, course_id):
+        return True
```

Comment:
Probably can mock has_access so that this function always returns true in your test.